### PR TITLE
Documentation improvements and course materials templates

### PR DESCRIPTION
**Pull request by brymz**
Continues from #417 with resolved merge conflict.

- [x] Address `language:` condition for `schedule.md` in `docs/` (as per #416)
- [x] Revise feedback links for course and site content maintainers (as per #409)
- [x] add raw `templates/` links
- [x] add `contact us`

### CODE REVIEW

**Comment by brymz** [comment]
This PR is ready for review. As the notes for the PR have gotten a bit hidden, I'll summarize:

The first attempt at a general course template (#377 & #408) was abandoned in favor of a `docs/` only approach (#415 & #417). PR #417 became conflicted and was resolved through this PR #419.

Changes Summary
- `docs/` was split into `course/`, `site/`, and `templates/` 
- Most of the original `docs/` were moved to `course/` and revised or replaced with better focus (e.g., `course-customization.md`)
- `site/` documentation was added for site developers' reference to course structure and customization revisions, including a `get-site-template.md`
- `templates/` include comments for managing files given the `course-structure`
- Feedback links in `audience.md` and `contributing.md` were revised for multiple repos (#409), including a `Contact Us` set of links in `nav/about`

File: docs/site/course-structure.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+- `START-for-self-guided-students.md`: Anyone should be able to start from here
+
+#### Course / Workshop
+- `computer-setup.md`: List preparations for prerequisite knowledge & technology 
+- `syllabus.md`: Course details and policy. 
+
+### Layout and Linkages
+ 
+```
+    START-for-sef-guided-students.md        syllabus.md
```
Comment (ethanwhite) [inline docs/site/course-structure.md]: self

**Comment by ethanwhite** [comment]
Really great work @brymz. I flagged one typo I noted on my way through. Let me know when that's fixed and we'll get this in.

File: docs/site/course-structure.md (hunk 1 above)
Comment (brymz) [inline docs/site/course-structure.md]: Done.

**Comment by brymz** [comment]
Thanks for the positive feedback, @ethanwhite. This is ready to merge.

**Pull request merged**